standard/docs/en/extensions/index.md

```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
 
 ## Using extensions
 
-Extensions are applied by adding their URLs to the `extensions` array in the release or record package. To use one or more extensions, select them from the extension lists below and include the following in your release or record package metadata:
-
-<style><!--
-#using-extensions div[class^='highlight'] pre {
-   white-space: pre-wrap;  
-   white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
-   white-space: -pre-wrap;      /* Opera 4-6 */
-   white-space: -o-pre-wrap;    /* Opera 7 */
-   word-wrap: break-word;       /* Internet Explorer 5.5+ */
-}
---></style> 
-
-```json
-{
-    "extensions":[],
-    "releases":[]
-}
-```
+Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions and read their documentation using the [Extension Explorer](https://extensions.open-contracting.org/en/).
```

Comment:
It's not clear from this page what an extension's URL is. We should either include a static example of how an extension is declared, explain that it is the URL of the `extension.json` file or update the text to make it clear the URLs can be found from the extension explorer:

> Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions, read their documentation and find the URL to add using the [Extension Explorer](https://extensions.open-contracting.org/en/).

Reply:
For now I've copied your paragraph in 8b52d52. This (and other text through the docs) can be improved in 1.1.5; I just copied and moved the first sentence from a (now deleted) page of 1.1.3.